pwndbg/color/memory.py

```diff
@@ -24,6 +25,28 @@
 )
 
 
+# Ex: convert and colorize address 0x7ffff7fcecd0 to string `0x7ffff7fcecd0 (_dl_fini)`
+# If no symbol, it is omitted
+def get_address_and_symbol(address: int) -> str:
+    symbol = pwndbg.gdblib.symbol.get(address) or None
+    if symbol:
+        symbol = f"{address:#x} ({symbol})"
+    return get(address, symbol)
+
+
+# Ex: convert and colorize address to symbol if it can be resolved, else return hex string of address
+def get_address_or_symbol(address: int) -> str:
+    return attempt_colorized_symbol(address) or get(address)
+
+
+# Convert address to colorized symbol (if symbol is there), else None
+def attempt_colorized_symbol(address: int) -> str | None:
+    symbol = pwndbg.gdblib.symbol.get(address) or None
+    if symbol:
+        return get(address, symbol)
+    return None
+
+
 def get(address: int | gdb.Value, text: str | None = None, prefix: str | None = None) -> str:
```

Comment:
I'm surprised to see type union syntax in existing code, we support Python 3.8 which doesn't support this syntax...